src/crate/crash/test_command.py

```diff
@@ -21,6 +22,23 @@
 from distutils.version import StrictVersion
 
 
+@contextmanager
```

Comment:
I know this could be achieved by adding `__enter__` and `__exit__` methods to the CrateShell class, but I am reluctant to do so, as this is only for suppressing the warning in the test suite.

Reply:
What's speaking against supporting the contextmanager generally?

Reply:
As I said, it's mainly for testing. However, I don't have a strong opinion.

Reply:
I think it could generally be useful, so I'd be in favour of adding it to the `CrateShell` class.